pyproject.toml

```diff
@@ -10,7 +10,7 @@ authors = []
 
 [tool.poetry.dependencies]
 python = "^3.8.1"  # ^3.8.1 required by flake8
-ops = "^2.6.0"
+ops = "^2.0.0"
```

Comment:
JFMI, what is the reason for downgrade min version here?

Reply:
downgraded by mistake - all other ops versions were `2.0.0` (unit, integration, etc). upgraded to `2.8.0` in ee4b09862143abb00c36232590ac72f383d91e5b to be able to use `unit.set_ports()`